### DISCUSSION TITLE
Great addon!

### DISCUSSION THREAD

**Discussion by Grizz2202** [Nixxen/StealYourCarbon#5] [General]
This thing has worked like a champ for 2 expansions. Are we going for Wrath too? :)

**Comment by Nixxen** [comment]
That's great to hear! Glad to hear more people are using it as well 😎

I haven't played since Classic, so I haven't kept the pulse on how things have been working out when it comes to stability, but there haven't been any reports of breaks *knocks on wood*

If it breaks, just let me know and I'll see if I can have it fixed within some reasonable time.

**Reply by Grizz2202** [reply]
hi nixxen it's me again! you said to let you know if it broke, and it looks like they changed the way you poll for bag slots (or something). is there any way you could look into this? i know about half+ of my guild would appreciate it because ive made them all download this lol! 

this is what im getting in the bugsack, it looks like something with "GetContainerNumSlots' (maybe they changed the syntax/function):

1x StealYourCarbon/Config_Bags.lua:121: attempt to call global 'GetContainerNumSlots' (a nil value)
[string "@StealYourCarbon/Config_Bags.lua"]:121: in function `UpdateBagList'
[string "@StealYourCarbon/Config_Bags.lua"]:109: in function `GenerateBagsPanel'
[string "@StealYourCarbon/Config.lua"]:24: in function <StealYourCarbon/Config.lua:14>
[string "=[C]"]: in function `Show'
[string "@Interface_Wrath/FrameXML/UIParent.lua"]:2214: in function `SetUIPanel'
[string "@Interface_Wrath/FrameXML/UIParent.lua"]:2020: in function `ShowUIPanel'
[string "@Interface_Wrath/FrameXML/UIParent.lua"]:1927: in function <Interface_Wrath/FrameXML/UIParent.lua:1923>
[string "=[C]"]: in function `SetAttribute'
[string "@Interface_Wrath/FrameXML/UIParent.lua"]:2733: in function `ShowUIPanel'
[string "@Interface_Wrath/FrameXML/UIParent.lua"]:2758: in function `ShowOptionsPanel'
[string "*GameMenuFrame.xml:92_OnClick"]:2: in function <[string "*GameMenuFrame.xml:92_OnClick"]:1>

**Reply by Nixxen** [reply]
Hey Grizz!

I currently do not have the game installed (honestly, not even a gaming rig set up at the moment), so it's hard to verify what works or does not work.

I did have a glance around to see what other mods changed and saw some changes to the `GetContainerNumSlots`. I'll need to have a more thorough look at what actually has changed, and how these mods solved this.

If I were to make some tests and set up an alpha branch here when I get the time to look into it - Would you be able to test the changes to make sure they run?

**Comment by Grizz2202** [comment]
definitely! i have this set to go to my email on my phone, so i get these messages pretty quickly

**Reply by Nixxen** [reply]
I've made a theoretical quick fix for the issue. I suspect there are other problems as well (such as on-load, or bank slots), but want to test first to see if this solves the original error message.
Experimental branch can be found at https://github.com/Nixxen/StealYourCarbon/tree/6-attempt-to-call-global-getcontainernumslots-a-nil-value

**Comment by Grizz2202** [comment]
i think that did it, not getting an error!

**Reply by Nixxen** [reply]
Excellent! Did you also try opening the bank to see if any errors popped up?

**Reply by Nixxen** [reply]
Assuming this has been working fine since I have not heard back. I'll merge the changes into the master branch.
